roles/ipaserver/README.md

```diff
@@ -168,6 +168,22 @@ Server installation step 2: Copy `<ipaserver hostname>-chain.crt` to the IPA ser
 
 The files can also be copied automatically: Set `ipaserver_copy_csr_to_controller` to true in the server installation step 1 and set `ipaserver_external_cert_files_from_controller` to point to the `chain.crt` file in the server installation step 2.
 
+Since version 4.10, FreeIPA suppots creating certificates using random serial numbers. This can only be activated on new domain deployments and once present, will be also activated on replicas. An example of an inventory file to deploy a server with random serial numbers enabled is:
```

Comment:
There is a typo: suppots

Reply:
Please rephrase the sentence "This can only .. will be also activated on replicas.". It should more state that this is a feature that can only be activated with the first server deployment and is then forever activated for the whole domain.

Reply:
Thanks, the new version is crystal clear now.